docs/source/user-guide/sql/explain.md

```diff
@@ -0,0 +1,69 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# EXPLAIN
+
+The `EXPLAIN` command shows the logical and physical execution plan for the specified SQL statement.
+
+<pre>
+EXPLAIN [ANALYZE] [VERBOSE] statement
+</pre>
+
+## EXPLAIN
+
+Shows the execution plan of a statement.
```

Comment:
Could we mention the info that `verbose` adds?

Reply:
I mentioned `VERBOSE` at [e9cf987](https://github.com/apache/arrow-datafusion/pull/3767/commits/e9cf98756625e5ec0516dbe6fab33de56c2895cb). 
please take a look.